EIPS/eip-4886.md

```diff
@@ -0,0 +1,292 @@
+
+The register is a single smart contract that stores all nomination and register records. The information held for each is as follows:
+ * Nomination:
+    * The address of the Nominator
+    * The address of the Proposed Proxy
+
+* Proxy Record:
+    * The address of the Nominator
+    * The address of the Proxy
+    * The delivery address for proxied deliveries
+
+Any address can act as a Nominator or a Proxy. A Nomination must have been made first in order for an address to accept acting as a Proxy. 
+
+A Nomination cannot be made to an address that is already active as either a Proxy or a Nominator, i.e. that address is already in an active proxy relationship.
+
+The information for both Nominations and Proxy records is held as a mapping. For the Nomination this is address => address for the Nominator to the Proxy address. For the Proxy Record the mapping is from address => struct for the Proxy Address to a struct containing the Nominator and Delivery address.
+
+Mapping between an address and its Nominator and Delivery address is a simple process as shown below:
+
+    Contract / Dapp                        Register
+
+      |                                       |
+      |------------- 0x4567..---------------> |
+      |                                       |
+      | <-------nominator: 0x1234..---------- |
+      |         delivery: 0x9876..            |
+      |                                       |
+
+The protocol is fully backwards compatible. If it is passed an address that does not have an active mapping it will pass back the received address as both the Nominator and Delivery address, thereby preserving functionality as the address is acting on its own behalf.
+
+    Contract / Dapp                        Register
+
+      |                                       |
+      |------------- 0x0222..---------------> |
+      |                                       |
+      | <-------nominator: 0x0222..---------- |
+      |         delivery: 0x0222..            |
+      |                                       |
+
+If the EPS register is passed the address of a Nominator it will revert. This is of vital importance. The purpose of the proxy is that the Proxy address is operating on behalf of the Nominator. The Proxy address therefore can derive the same benefits as the Nominator (for example discord roles based on the Nominator's holdings, or mint NFTs that require another NFT to be held). It is therefore imperative that the Nominator in an active proxy cannot also interact and derive these benefits, otherwise two addresses represent the same holding. A Nominator can of course delete the Proxy Record at any time and interact on it's own behalf, with the Proxy address instantly losing any benefits associated with the proxy relationship.
+
+## Solidity Interface Definition
```

Comment:
This isn't a valid EIP section.  Consider making it a subsection of the `## Specification` instead.

Reply:
Also done. Thank you.